indico/modules/events/timetable/templates/display/indico/_subcontribution.html

```diff
@@ -4,10 +4,16 @@
 
 
 {% macro render_subcontribution(subcontrib, event, show_notes=true) %}
-    <li class="timetable-subcontrib timetable-item">
+    {% set contrib = subcontrib.contribution %}
+    {% set anchor -%}
+        sc-{{ contrib.friendly_id }}-
+        {{- subcontrib.friendly_id }}-{{ subcontrib.title | slugify(maxlen=30) }}
```

Comment:
I wonder if it'd make sense to accept `*values` in slugify so you could do this:

    {% set anchor = slugify('sc', contrib.friendly_id, 'subcontrib.friendly_id', subcontrib.title, maxlen=30) %}